setup.py

```diff
@@ -187,21 +187,7 @@ def run(self):
     description="LINE Messaging API SDK for Python",
     long_description=long_description,
     license='Apache License 2.0',
-    packages=[
-        "linebot",
-        "linebot.models",
-        "linebot.v3",
-        "linebot.v3.audience",
-        "linebot.v3.insight",
-        "linebot.v3.liff",
-        "linebot.v3.messaging",
-        "linebot.v3.models",
-        "linebot.v3.module",
-        "linebot.v3.moduleattach",
-        "linebot.v3.oauth",
-        "linebot.v3.shop",
-        "linebot.v3.webhooks",
-    ],
+    packages=find_packages(include=["linebot*"]),
```

Comment:
It's same as `['linebot', 'linebot.constants', 'linebot.models', 'linebot.v3', 'linebot.v3.oauth', 'linebot.v3.moduleattach', 'linebot.v3.module', 'linebot.v3.shop', 'linebot.v3.insight', 'linebot.v3.models', 'linebot.v3.audience', 'linebot.v3.webhooks', 'linebot.v3.messaging', 'linebot.v3.liff', 'linebot.v3.oauth.models', 'linebot.v3.oauth.api', 'linebot.v3.moduleattach.models', 'linebot.v3.moduleattach.api', 'linebot.v3.module.models', 'linebot.v3.module.api', 'linebot.v3.shop.models', 'linebot.v3.shop.api', 'linebot.v3.insight.models', 'linebot.v3.insight.api', 'linebot.v3.audience.models', 'linebot.v3.audience.api', 'linebot.v3.webhooks.models', 'linebot.v3.webhooks.api', 'linebot.v3.messaging.models', 'linebot.v3.messaging.api', 'linebot.v3.liff.models', 'linebot.v3.liff.api']`